tests/projects/test_databricks.py

```diff
@@ -251,12 +251,15 @@ def test_run_databricks(runs_submit_mock, runs_get_mock, cluster_spec_mock, set_
 
 @pytest.mark.usefixtures("before_run_validations_mock", "runs_cancel_mock",
                          "dbfs_mocks", "cluster_spec_mock", "set_tag_mock")
+@pytest.mark.parametrize("runtime_version", [
+    '5.5.x-scala2.11', '6.6.x-scala2.11', '7.0.x-cpu-ml-scala2.12', 'latest-experimental-scala2.12'
```

Comment:
Does this patch depend on the runtime version? Or is it already parametrized to test other stuff?